recipe/conda_build_config.yaml

```
@@ -6,3 +6,12 @@ numpy:
   - 1.16
 MACOSX_SDK_VERSION:            # [osx and x86_64]
   - 11.0                       # [osx and x86_64]
+build_type:
+  - release
+  - debug
+channel_targets:
+  - conda-forge main
+  - conda-forge debug
+zip_keys:
+  - build_type
+  - channel_targets
```

This has to be a list of lists.
```suggestion
zip_keys:
  - - build_type
    - channel_targets
```